source/triggers/aws-eventbridge.txt

```diff
@@ -152,6 +164,238 @@ Procedure
+                                 "extended_json_enabled": <boolean>
+                              }
+                           }
+                        },
+                        "error_handler": {
+                           "config": {
+                              "enabled": <boolean>,
+                              "function_name": "<Error Handler Function Name>"
+                           }
+                        }
+                     }
+            
+            For more information on trigger configuration files, see
+            :ref:`Trigger Configuration Files <appconfig-triggers>`.
+
+.. _eventbridge-error-handler-parameters:
+
+Error Handler Parameters
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default error handler has two parameters: ``error`` and ``changeEvent``.
+
+``error`` 
+`````````
+Has two attributes: ``code`` and ``message``.
+
+- ``code``: For a list of error codes 
+used by the error handler, see the below section on :ref:`Error 
+Codes <eventbridge-error-codes>`. 
+
+- ``message``: The unfiltered error message from an errored Eventbridge
+put request.
+
+``changeEvent``
+```````````````
+
+The requested change to your data made by EventBridge. For more information
+on types of change events and their configurations, see 
+:manual:`Change Events </reference/change-events>`.
+
+.. _eventbridge-error-codes:
+
+Error Codes
+~~~~~~~~~~~
+
+If an error was recevied from EventBridge, the event processor will parse the 
+error as either ``DOCUMENT_TOO_LARGE`` or ``OTHER``. This parsed error is passed
+to the error handle function. 
+
+``DOCUMENT_TOO_LARGE``
+``````````````````````
+If the put entry for an EventBridge trigger event is larger
+than 256 KB, Eventbridge will throw an error. The error will either contain:
+
+1.  `status code: 400 <https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400>`_ and 
+``total size of the entries in the request is over the limit``.
+
+2. `status code: 413 <https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413>`_, 
+which indicates a too large payload. 
+
+``OTHER``
+`````````
+The default bucket for all other errors.
+
+Error Handler Logs
+~~~~~~~~~~~~~~~~~~
+
+You can view error logs for your Event Bridge Trigger error handler in the 
+application logs. 
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: ui
+
+      1. Click ``Logs`` in the left navigation of the App Services UI.
+      2. Click the :guilabel:`Filter by Type` dropdown and select
+      :guilabel:`Error Handlers` to view logs for the Trigger error handler.
+
+   .. tab::
+      :tabid: cli
+
+      Use the ``error_handler`` ``--type`` flag to 
+      view logs for the Trigger error handler.
+
+      .. code-block:: shell
+         
+         {+cli-bin+} logs list --type=error_handler
+
+
+To learn more about viewing application logs,
+see :ref:`View Application Logs <view-logs>`.
+
+Optimization
+~~~~~~~~~~~~
+
+By keeping track of the most common error messages you receive in ``error.message``,
```

Comment:
I think it's more that the recommendation is to track all of the error messages received, then make the special cases for the most common ones. 
Also, what are the different error messages that can be received? is this referring to optimizing all of the myriad messages thrown in the `OTHER` error bucket?

Reply:
Updated copy. Will ask Taylor about including list of common error messages.